postgres_toast/toast-value-store/src/main/java/io/debezium/demos/pgtoast/JsonObjectSerde.java

```diff
@@ -0,0 +1,91 @@
+package io.debezium.demos.pgtoast;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Map;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serializer;
+
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+/**
+ * A {@link Serde} that (de-)serializes JSON.
+ */
+@RegisterForReflection
+public class JsonObjectSerde implements Serde<JsonObject> {
```

Comment:
This is a cross-cutting concern that should be added into Debezium core.

Reply:
Yes, I was thinking about it, too. But than it's even more generic than Debezium. Unless we make it actually change-event aware and somehow expose "before" and "after" as dedicated objects for instance.

Reply:
I would not mind to make it Debezium specific so it is easy to consume.

Reply:
Logged as https://issues.jboss.org/browse/DBZ-1533